src/auth/crud.py

```diff
@@ -92,7 +75,8 @@ async def task_clean_expired_user_sessions(db_session: AsyncSession):
     await db_session.commit()
 
 
-async def user_info(db_session: AsyncSession, session_id: str) -> None | dict:
```

Comment:
I think I disagree a little here; it might not communicate anything specific, but it communicates more than just the noun "user_info". The function is a function not a data type, so i dont think it should be named like so; if you'd rather functions be more specific then I'm actually a fan of naming things as specific as possible, e.g.: get_site_user_with_session_id

Reply:
Regardless of which name we prefer (not a very important argument imo), changing them just for the sake of it increases the chance that we accidentally introduce a bug and makes things more unfamiliar.

If you'd like we can have a separate discussion about naming conventions later & include it all here https://github.com/CSSS/csss-site-backend/wiki/Style-Guide !

> My original motivation is that "get" doesn't have a lot of meaning. For example in 
> 
> ```py
> info = crud.get_user_info()
> # vs
> info = crud.user_info()
> ```
> 
> The second is cleaner than the first, because when the function is exactly what it returns. We don't need the name to communicate that it returns a value, because the function is clearly returning a value. I agree that being more specific is often a good thing, but there's a reasonable limit. `get_site_user_with_session_id` is definitely too much, unless there's some kind of motivation / pattern it's following. (ie. we have a 5 different functions that get site user using the same pattern)